CHANGELOG.md

```diff
@@ -13,6 +13,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+## [1.5.1] - 2023-05-02
```

Comment:
I'm no expert on versioning... Would this be v1.6 since functionality was added?  I can live with either one, just wasn't sure.